react/features/prejoin/components/native/Prejoin.tsx

```diff
@@ -149,16 +163,14 @@ const Prejoin: React.FC<IPrejoinProps> = ({ navigation }: IPrejoinProps) => {
                 }
                 <Button
                     accessibilityLabel = 'prejoin.joinMeeting'
-                    disabled = { joinButtonDisabled }
                     labelKey = 'prejoin.joinMeeting'
-                    onClick = { isJoining ? undefined : onJoin }
+                    onClick = { isJoining ? undefined : maybeJoin }
                     style = { styles.joinButton }
                     type = { PRIMARY } />
                 <Button
                     accessibilityLabel = 'prejoin.joinMeetingInLowBandwidthMode'
-                    disabled = { joinButtonDisabled }
                     labelKey = 'prejoin.joinMeetingInLowBandwidthMode'
-                    onClick = { onJoinLowBandwidth }
+                    onClick = { isJoining ? undefined : onJoinLowBandwidth }
```

Comment:
Looks like the low BW join is not protected by the display name check.

Reply:
What do you mean? It calls the same method that eventually displays the prompt.

Reply:
Nevermind, I misread!